packages/atomic/src/components/common/search-box/submit-button.tsx

```diff
@@ -12,20 +12,25 @@ export const SubmitButton: FunctionalComponent<Props> = ({
   onClick,
   ...defaultButtonProps
 }) => (
-  <Button
-    style="primary"
-    class="-my-px -mr-px flex h-auto w-12 shrink-0 items-center justify-center rounded-l-none rounded-r-md"
-    part="submit-button"
-    ariaLabel={bindings.i18n.t('search')}
-    onClick={() => {
-      onClick?.();
-    }}
-    {...defaultButtonProps}
+  <div
+    part="submit-button-wrapper"
+    class="mr-2 flex items-start items-center justify-center py-2"
```

Comment:
This is from an old PR but `items-start` and `items-center` change the same CSS value. We can't have both of these. 

<img width="648" alt="image" src="https://github.com/user-attachments/assets/dcdc56f3-603d-4d0c-9c81-e6869d46f7fa">